docs/conf.py

```diff
@@ -4,7 +4,7 @@
 # Package information
 project = "Pymanopt"
 author = "Jamie Townsend, Niklas Koep, Sebastian Weichwald"
-copyright = "2016-2020, {:s}".format(author)
+copyright = "2016-2021, {:s}".format(author)
```

Comment:
We could do
```
from datetime import date
copyright = f"2016–{date.today().year}, {author:s}"
```
?

Reply:
Yep, I like it!

Reply:
https://github.com/pymanopt/pymanopt/pull/146/commits/c0daf6e2c549e2857f8acfc69ae97635705a5342